subprojects/shipkit/src/main/groovy/org/shipkit/gradle/release/ReleaseNeededTask.java

```diff
@@ -84,6 +85,20 @@ public void setPullRequest(boolean pullRequest) {
         this.pullRequest = pullRequest;
     }
 
+    /**
+     * If the comparison of publications should be skipped.
```

Comment:
Can you write when this property is useful? (the use case)

It really helps understand the feature, if the user can read and understand the reason why he might use it!

Reply:
yes, indeed. I will add a proper use case.